consumer.go

```diff
@@ -181,15 +183,13 @@ func convertFromRetryableMessage(message kcronsumer.Message) Message {
 
 func (c *consumer) consume() {
 	c.logger.Debug("Consuming is starting")
-	c.wg.Add(1)
```

Comment:
I think we should not delete this wait group integration on consume method. When Konsumer Stop called we are waiting for all go routines to finish their work by c.wg.Wait()